code/datums/status_effects/debuffs.dm

```diff
@@ -371,3 +371,13 @@
 	REMOVE_TRAIT(owner, TRAIT_MUTED, TRAIT_STATUS_EFFECT(id))
 	return ..()
 
+/datum/status_effect/spacefreeze
+	id = "spacefreeze"
+
+/datum/status_effect/spacefreeze/on_creation(mob/living/new_owner)
+	. = ..()
+	to_chat(new_owner, span_danger("The cold vacuum instantly freezes you, maybe this was a bad idea?"))
```

Comment:
"Cold vacuum" is an oxymoron.

Reply:
why would it be? space has a temperature, it happens to be cold

Reply:
Space is nothing, it literally can't be cold, because it's nothing.

Reply:
You can define temperature stricly out of radiations. And if you consider this temperature, space is cold